### ISSUE TITLE
Support cache pruning

### ISSUE THREAD

**Issue by mcassaniti** [juicedata/juicefs#4574]
**What would you like to be added**: The local disk cache is periodically pruned of chunks that no longer exist in object storage. At the minimum a command that supports manual pruning of the cache is what I would like.

**Why is this needed**: The local disk cache can grow beyond the size of the object storage. As an example, if the S3 bucket that backs a JuiceFS file system is 30GB in size and `juicefs warmup` is run daily, the local disk cache will likely grow beyond 30GB over time.

I do not want to flush the entire cache and re-create it which would be quite wasteful. I also don't want to build my own cache pruning tool if I can avoid it.

**Comment by SandyXSD** [comment]
It's a bit too heavy to prune cache blocks based on their existence in the object storage. Usually the cache blocks will be deleted as well when a client removes objects, while blocks on other nodes are kept though.

To periodically prune cache blocks, the mount option `cache-expire` might be helpful.

**Comment by mcassaniti** [comment]
I see that `cache-expire = 2-random` is the default value. Rather than me make assumptions on what that means can you let me know?

**Comment by SandyXSD** [comment]
No, that's `cache-eviction = 2-random`. The `cache-expire` option is used to specify the survival time of cache blocks.

**Comment by SandyXSD** [comment]
Well, I forgot that it's not supported in the v1.1 releases yet. For now you may check usage of the `cache-expire` option on the main branch; and it will be added in v1.2.

**Comment by SandyXSD** [comment]
Will be fixed at v1.2.
